src/data_cleaning.R

```diff
@@ -0,0 +1,54 @@
+#preamble
+rm(list=ls()) # clear vars
+cat("\014") # clear console
+library(tidyverse)
+library(lubridate)
+library(ggplot2)
+library(tsibble)
+
+
+setwd("C:\\Users\\Yamni\\Desktop\\TNMD-mini-hackathon\\air_quality_data")
+
+
+#read data
+data<-read.csv("Air_Quality_Monitoring_Data.csv")
+
+#data cleaning
+
+#Quick eyeballing looks like the data is hourly for the three different locations that we have.
+#Going to split up the Date Time column and create some order.
+data<- data %>% 
+      mutate(DateTime2= dmy_hms(DateTime, tz= "Australia/ACT")) %>% #Thanks Mark!
+  arrange(DateTime2)
+
+# Creating three different datasets for each of the three locations as they seem to start at different times
+
+# Some initial exploratory code
+#data_monash<- data%>% filter(Name=="Monash") 
+#tried converting to tsibble but it looks like there are duplicated values. Looking at these
+#duplicates_df<- duplicates(data_monash%>% drop_na()) #this generates a df that has 22 rows 50 of them on 13/07/2022 at 6PM and 2 at 28/07/2022 at 7pm.
+# As a general rule, I am going to only select unique rows.
+
+data_monash<- data %>% 
+  filter(Name=="Monash") %>% 
+  distinct(DateTime2, .keep_all=TRUE) %>% # This is an inbuilt R function. Not sure which value it actually chooses.
```

Comment:
According to the [documentation](https://www.rdocumentation.org/packages/dplyr/versions/0.7.8/topics/distinct), the it picks the first row.